core/src/main/java/com/radixdlt/RadixNodeModule.java

```diff
@@ -200,34 +201,30 @@ protected void configure() {
         properties.get(
             "mempool.relayer.max_message_payload_size",
             MempoolRelayerConfig.DEFAULT_MAX_MESSAGE_PAYLOAD_SIZE);
-    if (mempoolRelayerMaxPeers <= 0) {
-      throw new IllegalArgumentException(
-          "Invalid configuration: mempool.relayer.max_peers ("
-              + mempoolRelayerMaxPeers
-              + ") must be a non zero positive number.");
-    }
-    if (mempoolRelayerMaxMessageTransactionCount <= 0) {
-      throw new IllegalArgumentException(
-          "Invalid configuration: mempool.relayer.max_message_transaction_count ("
-              + mempoolRelayerMaxMessageTransactionCount
-              + ") must be a non zero positive number.");
-    }
-    if (mempoolRelayerMaxMessagePayloadSize < NodeConstants.DEFAULT_MAX_TRANSACTION_SIZE) {
-      throw new IllegalArgumentException(
-          "Invalid configuration: mempool.relayer.max_message_payload_size ("
-              + mempoolRelayerMaxMessagePayloadSize
-              + ") must be at least the maximum transaction size ("
-              + NodeConstants.DEFAULT_MAX_TRANSACTION_SIZE
-              + ").");
-    }
-    if (mempoolRelayerMaxRelayedSize < mempoolRelayerMaxMessagePayloadSize) {
-      throw new IllegalArgumentException(
-          "Invalid configuration: mempool.relayer.max_relayed_size ("
-              + mempoolRelayerMaxRelayedSize
-              + ") must be at least mempool.relayer.max_message_payload_size ("
-              + mempoolRelayerMaxMessagePayloadSize
-              + ").");
-    }
+    Preconditions.checkArgument(
+        mempoolRelayerMaxPeers <= 0,
+        "Invalid configuration: mempool.relayer.max_peers ("
+            + mempoolRelayerMaxPeers
+            + ") must be a non zero positive number.");
+    Preconditions.checkArgument(
+        mempoolRelayerMaxMessageTransactionCount <= 0,
+        "Invalid configuration: mempool.relayer.max_message_transaction_count ("
+            + mempoolRelayerMaxMessageTransactionCount
+            + ") must be a non zero positive number.");
+    Preconditions.checkArgument(
+        mempoolRelayerMaxMessagePayloadSize < NodeConstants.DEFAULT_MAX_TRANSACTION_SIZE,
+        "Invalid configuration: mempool.relayer.max_message_payload_size ("
+            + mempoolRelayerMaxMessagePayloadSize
+            + ") must be at least the maximum transaction size ("
+            + NodeConstants.DEFAULT_MAX_TRANSACTION_SIZE
+            + ").");
+    Preconditions.checkArgument(
+        mempoolRelayerMaxRelayedSize < mempoolRelayerMaxMessagePayloadSize,
+        "Invalid configuration: mempool.relayer.max_relayed_size ("
+            + mempoolRelayerMaxRelayedSize
+            + ") must be at least mempool.relayer.max_message_payload_size ("
+            + mempoolRelayerMaxMessagePayloadSize
+            + ").");
```

Comment:
The `.checkArgument()` is convenient because it does formatting.
So you can do the multiline java string (`""" ...`) with placeholders (`%s`) - and it will also avoid rendering, as a bonus.